ui/tests/helpers/module-for-job.js

```diff
@@ -30,19 +31,28 @@ export default function moduleForJob(title, context, jobFactory, additionalTests
     });
 
     test('visiting /jobs/:job_id', async function(assert) {
-      assert.equal(
-        currentURL(),
-        urlWithNamespace(`/jobs/${encodeURIComponent(job.id)}`, job.namespace)
+      const expectedURL = new URL(
+        urlWithNamespace(`/jobs/${encodeURIComponent(job.id)}`, job.namespace),
+        window.location
       );
+      const gotURL = new URL(currentURL(), window.location);
+
+      assert.deepEqual(gotURL.path, expectedURL.path);
+      assert.deepEqual(gotURL.searchParams, expectedURL.searchParams);
       assert.equal(document.title, `Job ${job.name} - Nomad`);
     });
 
     test('the subnav links to overview', async function(assert) {
       await JobDetail.tabFor('overview').visit();
-      assert.equal(
-        currentURL(),
-        urlWithNamespace(`/jobs/${encodeURIComponent(job.id)}`, job.namespace)
+
+      const expectedURL = new URL(
+        urlWithNamespace(`/jobs/${encodeURIComponent(job.id)}`, job.namespace),
+        window.location
       );
+      const gotURL = new URL(currentURL(), window.location);
+
+      assert.deepEqual(gotURL.path, expectedURL.path);
+      assert.deepEqual(gotURL.searchParams, expectedURL.searchParams);
```

Comment:
I have no idea why these started fail 🤷 

The `currentURL` would have and extra query params for `sort`, so I'm parsing the URL to ignore order.

Reply:
This fails on periodic and parameterized jobs. Likely points to some more lurking bugs in how we handle periodic and parameterized jobs.